master/internal/stream/experiments.go

```diff
@@ -0,0 +1,277 @@
+package stream
+
+import (
+	"context"
+	"database/sql"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/uptrace/bun"
+
+	"github.com/determined-ai/determined/master/internal/db"
+	"github.com/determined-ai/determined/master/pkg/model"
+	"github.com/determined-ai/determined/master/pkg/stream"
+)
+
+const (
+	// ExperimentsDeleteKey specifies the key for delete experiments.
+	ExperimentsDeleteKey = "experiments_deleted"
+	// ExperimentsUpsertKey specifies the key for upsert experiments.
+	ExperimentsUpsertKey = "experiment"
+)
+
+// ExperimentMsg is a stream.Msg.
+// determined:streamable
+type ExperimentMsg struct {
+	bun.BaseModel `bun:"table:experiments"`
+	// immutable attributes
+	ID        int           `db:"id" json:"id"`
```

Comment:
question: Does `db` do the same thing as `bun`?

Reply:
huh good point - I looked thru the docs and I don't think `db` is supported, but this has been working so not sure what's going on. I'll switch to `bun` though